scripts/package

```diff
@@ -13,6 +13,13 @@ TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
 
 cp ../bin/container-crontab .
-docker build -t ${REPO}/container-crontab:${TAG} .
 
-echo Built ${REPO}/container-crontab:${TAG}
+if dapper -d --build -f Dockerfile -- -t ${REPO}/container-crontab:${TAG}${SUFFIX} .; then
+    echo Built ${REPO}/container-crontab:${TAG}
+    elif [ "$?" != "42" ]; then
```

Comment:
more style then anything, is this the right way to indent? They are at the same level as the `if dapper... `

Reply:
Already changed the indent. Am i right? Thanks a lot.